src/mol-plugin-ui/viewport/simple-settings.tsx

```diff
@@ -55,12 +55,9 @@ const SimpleSettingsParams = {
         color: PD.Color(Color(0xFCFBF9), { label: 'Background', description: 'Custom background color' }),
         transparent: PD.Boolean(false)
     }, { pivot: 'color' }),
-    lighting: PD.Group({
-        renderStyle: Canvas3DParams.renderer.params.style,
-        occlusion: Canvas3DParams.postprocessing.params.occlusion,
-        outline: Canvas3DParams.postprocessing.params.outline,
-        fog: Canvas3DParams.cameraFog,
-    }, { pivot: 'renderStyle' }),
+    occlusion: Canvas3DParams.postprocessing.params.occlusion,
+    outline: Canvas3DParams.postprocessing.params.outline,
+    fog: Canvas3DParams.cameraFog,
```

Comment:
Any reason for moving these out of the ``lighting`` group? Seems like unnecessarily breaking old code.

Maybe just remove ``renderStyle`` instead?

Reply:
good point, will do